docs/contract.yaml

```diff
@@ -0,0 +1,145 @@
+openapi: 3.0.0
+info: 
+  title: Spec Math API Server
+  version: 1.0.0
+  description: An API that exposes some operations from the Spec Math Library
+servers:
+  - description: Spec Math API Server
+    url: http://specmath.cloud.google.com/v1
+paths:
+  /specMath/merge:
+    post:
+      description: perform a union on two specs, with optional defaults file and conflict resolutions
+      operationId: specMathUnion
+      requestBody:
+        description: data needed for the union operation
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/MergeBody"
+      responses: 
+        '200':
+          description: operation was successful, returns the resultant spec 
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/OperationSuccess"
+        '406':
+          description: return information relevant to the conflict in case of a failed union
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  conflicts:
+                    type: array
+                    items:
+                      "$ref": "#/components/schemas/MergeConflictObject"
+        default:
+          description: error
+          content:
+            application/json:
+              schema:
+                "$ref":  "#/components/schemas/Error"  
+
+  /specMath/overlay:
+    post:
+      description: perform an overlay on a spec
```

Comment:
make sure difference between this and merge is clear

Reply:
also describe what it does

Reply:
addressed in 3309c33!